bot/exts/evergreen/trivia_quiz.py

```diff
@@ -111,23 +316,39 @@ async def quiz_game(self, ctx: commands.Context, category: str = None) -> None:
                         done_question.append(question_dict["id"])
                         break
 
-                q = question_dict["question"]
-                answer = question_dict["answer"]
+                if "dynamic_id" not in question_dict:
+                    question = question_dict["question"]
+                    answers = question_dict["answer"].split(", ")
+                else:
+                    format_func = DYNAMIC_QUESTIONS_FORMAT_FUNCS[question_dict["dynamic_id"]]
+
+                    question, answers = format_func(
+                        question_dict["question"],
+                        question_dict["answer"],
+                    )
+
+                    answers = [answers]
+
+                embed = discord.Embed(
+                    colour=discord.Colour.gold(),
+                    title=f"Question #{len(done_question)}",
+                    description=question,
+                )
+
+                if img_url := question_dict.get("image_url"):
+                    embed.set_image(url=img_url)
 
-                embed = discord.Embed(colour=discord.Colour.gold())
-                embed.title = f"Question #{len(done_question)}"
-                embed.description = q
                 await ctx.send(embed=embed)  # Send question embed.
 
             # A function to check whether user input is the correct answer(close to the right answer)
             def check(m: discord.Message) -> bool:
-                return (
-                    m.channel == ctx.channel
-                    and fuzz.ratio(answer.lower(), m.content.lower()) > 85
+                return m.channel == ctx.channel and any(
+                    fuzz.ratio(answer.lower(), m.content.lower()) > VARIATION_TOLERANCE
```

Comment:
Are you sure this will work well with numeric answers? `1023451` is pretty close to `1023145`, but not the right answer

Reply:
it will, since all numeric answers are pretty small, enough so that one little change will make it smaller than the ratio